### PR TITLE
Maximum bounds for the mimic in the overlay can now be specified.

### PR DESCRIPTION
**Pull request by apwilson**

### CODE REVIEW

File: sky/packages/sky/lib/widgets/mimic_overlay.dart [hunk 1]
```diff
@@ -10,18 +10,16 @@ import 'package:sky/widgets/basic.dart';
...
+  MimicOverlay({Key key, this.children, this.overlay,
+      this.duration: const Duration(milliseconds: 200), this.curve: linear,
+      this.mimicMaxBounds})
+      : super(key: key);
 
   List<Widget> children;
   GlobalKey overlay;
   Duration duration;
   Curve curve;
+  Rect mimicMaxBounds;
```
Comment (abarth) [inline sky/packages/sky/lib/widgets/mimic_overlay.dart]: Maybe call this `targetRect` ?

File: sky/packages/sky/lib/widgets/mimic_overlay.dart [hunk 2]
```diff
@@ -10,18 +10,16 @@ import 'package:sky/widgets/basic.dart';
...
-    Key key,
-    this.children,
-    this.overlay,
-    this.duration: const Duration(milliseconds: 200),
-    this.curve: linear
-  }) : super(key: key);
+  MimicOverlay({Key key, this.children, this.overlay,
+      this.duration: const Duration(milliseconds: 200), this.curve: linear,
+      this.mimicMaxBounds})
+      : super(key: key);
```
Comment (abarth) [inline sky/packages/sky/lib/widgets/mimic_overlay.dart]: IMHO, this was prettier before.  Now it just looks like a mess of stuff.

**Comment by abarth** [comment]
LGTM

File: sky/packages/sky/lib/widgets/mimic_overlay.dart (hunk 2 above)
Comment (apwilson) [inline sky/packages/sky/lib/widgets/mimic_overlay.dart]: I agree :(

**Pull request merged**